### DISCUSSION TITLE
understand-class-inheritance/

### DISCUSSION THREAD

**Discussion by giscus** [NachtgeistW/Berksey#91] [General]
# understand-class-inheritance/

JVM 这泛型比不上模板一点，连 C# 的都不如——残像  以下笔记摘自 D.N.Code 群聊天记录。 如有错误请指出。

https://blog.nachtgeistw.zone/understand-class-inheritance/

**Comment by c0nstexpr** [comment]
来踩了一下

**Comment by Trarizon** [comment]
**夜轮姐姐简直是天使，啊啊啊啊激动到跑圈！！！！**

**Reply by Phynon** [reply]
天使光轮！
